test/build/40_pull.bats

```diff
@@ -175,31 +175,37 @@ EOF
     # invalid character in image name
     cat <<'EOF' | image_ref_parse 'name*' 1
 error: image ref syntax, char 5: name*
+hint:  https://hpc.github.io/charliecloud/faq.html#how-do-i-specify-an-image-reference
```

Comment:
I recommend just one space after `hint:` because while it looks tidy here you could also plausibly also use the hint with other log levels.